docs/rustdoc-index.md

```diff
@@ -1,5 +1,62 @@
-# Fedimint technical reference documentation
+# Fedimint technical reference
 
 <!-- this page is used a landing page for https://docs.fedimint.org/ -->
+<!-- lots of links in this document are relative to the generated document, so disabling linkcheck altogether: -->
+<!-- markdown-link-check-disable -->
 
-This is documentation automatically generated from the Fedimint source code.
+## Target audience
+
+This documentation is automatically generated from the Fedimint source code,
+and it's meant for developers, builders and people who want to understand
+Fedimint project on an implementation level.
+
+If you are looking for higher level documentation and help, please check:
+
+* [Fedimint website](https://fedimint.org/)
+* [Fedimint github](https://github.com/fedimint/fedimint)
+* [Fedimint chat](https://chat.fedimint.org/)
+
+## State
+
+This is a recently added document and subject of continuous change. Please report problems and submit improvements.
+
+# Overview
+
+Fedimint is implemented in [Rust](https://www.rust-lang.org/) and consists of multiple modules (called "crates" in Rust).
```

Comment:
"modules" is a bit ambiguous in the Fedimint context, I didn't think you were talking about crates